materials/version-control-R.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: page
+element: notes
+title: Version Control
+language: R
+--- 
+
+> Remind students to setup a GitHub account and email me their username.
+
+> Arrange for partner to `push` following code for 'Collaborating' demo.
```

Comment:
"Arrange to have a teaching partner attend class and `push` the following code for the 'Collaborating' demo."